src/main/java/com/google/sps/data/DBUtils.java

```diff
@@ -31,6 +37,18 @@ public static Firestore db() {
     return database;
   }
 
+  public static CollectionReference users() {
+    return usersReference;
+  }
+
+  public static DocumentReference user(String userID) {
+    return usersReference.document(userID);
+  }
+
+  public static String getNestedPropertyName(String property, String nestedProperty) {
+    return property + "." + nestedProperty;
+  }
+
   public static CollectionReference recipes() {
     return recipesReference;
   }
```

Comment:
I just pulled the changes to DBUtils into my CommentPut branch, and I'm getting NullPointerExceptions on the return values of this function as well as comments()